### ISSUE TITLE
Remove `-d` flag option from push

### ISSUE THREAD

**Issue by ctlong** [cloudfoundry/cf-test-helpers#64]
This flag was deprecated and removed from the `cf push` command.

https://github.com/cloudfoundry/cf-test-helpers/blob/36ed5ab41b8fc386940381b9854a9375b6e986de/cf/push.go#L45-L46

**Comment by ctlong** [comment]
tldr; this issue was misguided and the PR resolving it introduced a bug into CATs. We should find a way to remove it from cf-test-helpers.

I thought that I was removing the `-d` flag from `cf push` with my [change](https://github.com/cloudfoundry/cf-test-helpers/commit/31eca81788e579390835aa22f8f4592bf580b1ab), as that flag was deprecated in cf CLI v7. However, the `Push` function in cf-test-helpers actually generates an application manifest and uses that manifest to `cf push` the application. Removing the `-d` option from the `Push` function removed the option to populate the routes field of the application manifest, which certain CATs isolation segment tests relied upon.

* https://cloudfoundry.slack.com/archives/C033ALST37V/p1664463227639259
* https://cloudfoundry.slack.com/archives/C01ABMVNE9E/p1664484810271339?thread_ts=1664482646.299389&cid=C01ABMVNE9E

**Comment by ctlong** [comment]
My proposal is to revert #71 and cut a patch release. Then to update the release descriptions to make it clear what happened.

**Comment by ctlong** [comment]
The revert commit is included in in the v2.2.0 release.
